2023-06-02 10:46:53,535 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 10:46:53,834 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 10:46:53,837 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 10:47:23,719 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 10:47:23,745 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 10:47:23,784 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 10:47:23,940 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 10:47:23,970 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 10:48:02,359 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 10:48:02,362 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 10:48:02,417 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 10:48:02,420 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 10:48:02,424 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 10:48:56,671 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 10:48:56,819 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 10:48:56,821 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 10:49:38,526 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 10:52:14,818 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 10:52:17,065 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 10:52:17,103 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 10:52:17,907 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 10:54:00,887 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 10:54:01,076 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 10:54:01,079 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 10:54:05,495 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 10:54:06,609 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 10:54:09,259 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:04:56,276 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:04:56,578 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:04:56,580 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 11:04:58,456 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:04:58,465 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:04:58,517 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:05:05,377 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:05:05,389 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 41\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:05:05,394 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","trace":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 41\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:05:49,069 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:05:49,213 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:05:49,215 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 11:05:53,008 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:05:53,017 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:05:53,074 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:06:21,266 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [67] - MESSAGE: Login ServiceLayer correcto
2023-06-02 11:06:51,221 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:06:51,370 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:06:51,372 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 11:06:57,989 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:06:58,010 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 11:06:58,053 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:06:58,151 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:06:58,182 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:07:00,897 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:07:00,900 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 11:07:00,935 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:07:00,939 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:07:00,941 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:07:01,442 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:07:01,445 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 11:07:01,482 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:07:01,485 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:07:01,488 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:07:10,617 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:07:10,620 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [19] - MESSAGE: Credenciales incorrectas  
2023-06-02 11:07:10,655 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:07:10,657 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.UnauthorizedAccessException","Message":"Token does not match","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2147024891,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:07:10,659 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Token does not match","trace":"   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 20\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:07:14,691 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:07:14,694 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:07:19,070 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:07:53,903 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:07:54,046 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:07:54,048 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 11:07:55,316 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:07:55,324 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:07:55,382 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:07:58,985 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:07:58,996 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:07:59,001 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","trace":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:08:02,853 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:08:02,856 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:08:02,858 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:08:04,065 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [67] - MESSAGE: Login ServiceLayer correcto
2023-06-02 11:08:50,172 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:08:50,176 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"Ha ocurrido un error","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:08:50,179 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Ha ocurrido un error","trace":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:09:38,608 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:09:38,756 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:09:38,758 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
2023-06-02 11:10:40,082 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:10:40,090 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:10:40,141 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:10:40,854 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:10:40,866 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:10:40,871 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","trace":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:10:45,743 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:10:45,746 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:10:45,749 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:10:51,742 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [67] - MESSAGE: Login ServiceLayer correcto
2023-06-02 11:10:53,385 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:10:53,390 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"Ha ocurrido un error: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:10:53,394 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Ha ocurrido un error: ","trace":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:11:32,496 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:11:32,500 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:11:32,503 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:11:32,773 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:11:32,777 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:11:32,780 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"No se ha podido hacer login a ServiceLayer | StatusCode: NotFound Content: ","trace":"   at ServiceLayer.doLogin() in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Classes\\ServiceLayer.cs:line 60\r\n   at SEI_WEBSERVICE.Controllers.PurchaseRequestController..ctor(ILogger`1 _logger) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 23\r\n   at lambda_method9(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass7_0.<CreateActivator>b__0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:11:39,122 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [16] - MESSAGE: Accediendo a Login
2023-06-02 11:11:39,124 INFO  SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [22] - MESSAGE: Login correcto
2023-06-02 11:11:39,127 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [45] - MESSAGE: Login ServiceLayer
2023-06-02 11:11:39,820 INFO  SEI_WEBSERVICE.Controllers.OrdenFabricacionController.doLogin [67] - MESSAGE: Login ServiceLayer correcto
2023-06-02 11:11:40,796 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [46] - MESSAGE: ERROR! -> Accediendo a ExceptionManager
2023-06-02 11:11:40,800 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [51] - MESSAGE: Raw exception: {"ClassName":"System.Exception","Message":"Ha ocurrido un error: ","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"SEI_BASE_WS_NET6","WatsonBuckets":null}
2023-06-02 11:11:40,804 ERROR SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.MoveNext [64] - MESSAGE: Return error: {"code":-3,"message":"Ha ocurrido un error: ","trace":"   at SEI_WEBSERVICE.Controllers.PurchaseRequestController.PostAsync(List`1 requestBody) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Controllers\\PurchaseRequest\\PurchaseRequestController.cs:line 67\r\n   at lambda_method5(Closure , Object )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at SEI_WEBSERVICE.Middlewares.AuthenticationMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\AuthenticationMiddleware.cs:line 24\r\n   at SEI_WEBSERVICE.Middlewares.ExceptionManagerMiddleware.Invoke(HttpContext context) in D:\\GitRepos\\SEI_BASE_WEBSERVICE_ASPNET6\\SEI_BASE_WS_NET6\\SEI_BASE_WS_NET6\\Middlewares\\ExceptionManagerMiddleware.cs:line 42"}
2023-06-02 11:30:05,717 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Application started. Press Ctrl+C to shut down.
2023-06-02 11:30:05,996 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Hosting environment: Development
2023-06-02 11:30:05,998 INFO  Microsoft.Hosting.Lifetime.OnApplicationStarted [0] - MESSAGE: Content root path: D:\GitRepos\SEI_BASE_WEBSERVICE_ASPNET6\SEI_BASE_WS_NET6\SEI_BASE_WS_NET6\
